src/components/Icon/ChevronRight.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 
-function ChevronRight() {
+function ChevronRight({ color }: { color?: string }) {
```

Comment:
I think it isn't necessary to have `ChevronLeft` and `ChevronRight`. We can have a `Chevron` component that receives the left or right orientation by parameter and that becomes the svg property `transform="rotate(0)"` and `transform="rotate(180)"`.

For example: 
![image](https://user-images.githubusercontent.com/45410089/175965126-bcc11f0d-08ff-420a-849b-69927eddbab0.png)